File: Dasar-dasar Pemrograman/Semester 1/Python/PR - Siswa Lulus.py
hasil_akhir = [
{'nama':'Reza', 'nilai':70},
{'nama':'Ciut', 'nilai':63},
{'nama':'Dian', 'nilai':80},
{'nama':'Badu', 'nilai':40}
]

def lulus_saja(param):
    for i in range(len(param)):
        if (param[i].get('nilai') > 65):
            print(param[i].get('nama'))

lulus_saja(hasil_akhir)

File: Dasar-dasar Pemrograman/Semester 1/PHP/PR - Siswa Lulus.php
<?php 

class Murid {
    private $nama;
    private $nilai;

    public function __construct($setNama, $setNilai) {
        $this->nama = $setNama;
        $this->nilai = $setNilai;
    }

    public function setNama($param) {
        $this->nama = $param;
    }

    public function setNilai($param) {
        $this->nilai = $param;
    }

    public function getNama() {
        return $this->nama;
    }

    public function getNilai() {
        return $this->nilai;
    }
}

$reza = new Murid('reza', 70);
$ciut = new Murid('ciut', 63);
$dian = new Murid('dian', 80);
$badu = new Murid('badu', 40);

$hasil_akhir = [$reza, $ciut, $dian, $badu];

function lulusAja($param) {
    foreach ($param as $p) {
        if ($p->getNilai() > 65) {
            echo $p->getNama()." ";
        }
    }
}

lulusAja($hasil_akhir);
